docs/conf.py

```diff
@@ -55,7 +55,12 @@
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {"https://docs.python.org/": None}
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None),
+    'numpy': ('https://docs.scipy.org/doc/numpy', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
+    'pandas': ('http://pandas.pydata.org/pandas-docs/stable/', None),
+    'astropy': ('http://docs.astropy.org/en/stable/', None)}
```

Comment:
```suggestion
    'astropy': ('http://docs.astropy.org/en/stable/', None),
    'plasmapy': ('http://docs.plasmapy.org/en/latest/', None),
}
```

Reply:
Great Caesar's ghost!  How could I have forgotten that?